packages/snap-toolbox/src/until/until.ts

```diff
@@ -0,0 +1,65 @@
+export type UntilOptions = {
+	checkMax: number;
+	checkCount: number;
+	checkTime: number;
+	exponential: number;
+	defer: boolean;
+	executeFunction: boolean;
+};
+
+const defaultUntilOptions: UntilOptions = {
+	checkMax: 20,
+	checkCount: 0,
+	checkTime: 50,
+	exponential: 1.1,
```

Comment:
I lowered this to `1.1` from `1.2`
With the defaults, the last interval with `1.1` is about 300ms compared to about 1900ms when it's `1.2`

Reply:
Maybe we should change some other defaults to get us around 5-7seconds?
```
1.1: 3150.124972129084
1.2: 11201.279977342421
```

Reply:
```
checkMax: 25,
checkTime: 60,
exponential: 1.1,
```
Yields: 6490.905922636333